schema.sql

```diff
@@ -7,5 +7,6 @@ CREATE TABLE animals (
     escape_attempts INT,
     neutered BOOLEAN NOT NULL,
     weight_kg DECIMAL,
+    species VARCHAR(40),
     PRIMARY KEY(id)
```

Comment:
- [ ] Adding the new column this way is not wrong however, it would be better if you could use the `ALTER` keyword to add a new column to a table.

See example:
```
ALTER TABLE animals ADD species varchar(40);
```